conformance/data_model/null.ion

```diff
@@ -0,0 +1,89 @@
+         (then (text "null.string"   ) (denotes (Null string   )))
+         (then (text "null.struct"   ) (denotes (Null struct   )))
+         (then (text "null.symbol"   ) (denotes (Null symbol   )))
+         (then (text "null.timestamp") (denotes (Null timestamp))))
+
+(ion_1_x "malformed nulls in text should signal an error"
+         (each (text "null.")
+               (text "null.1")
+               (text "null.ion")
+               (text "null.array")
+               (signals "invalid type")))
+
+(ion_1_x "in text, unquoted null cannot be used as an annotation"
+         (text "null::0")
+         (signals "cannot use unquoted keyword as an annotation"))
+
+(ion_1_x "in text, typed null suffix parsing has precedence over sexp operator rules"
+         (each (text "(null.)")
+               (text "(null.-)")
+               (text "(null..)")
+               (text "(null.true)")
+               (text "(null.1)")
+               (text "(null.nan)")
+               (text "(null.+inf)")
+               (text "(null.foo)")
+               (text "(null.[])")
+               (text "(null.())")
+               (text "(null.{})")
+               (signals "invalid type")))
+
+(ion_1_0 "null values in Ion 1.0 binary"
+         (then (binary 0x0F) (denotes (Null          )))
+         (then (binary 0x1F) (denotes (Null bool     )))
+         (then (binary 0x2F) (denotes (Null int      )))
+         (then (binary 0x3F) (denotes (Null int      )))
+         (then (binary 0x4F) (denotes (Null float    )))
+         (then (binary 0x5F) (denotes (Null decimal  )))
+         (then (binary 0x6F) (denotes (Null timestamp)))
+         (then (binary 0x7F) (denotes (Null symbol   )))
+         (then (binary 0x8F) (denotes (Null string   )))
+         (then (binary 0x9F) (denotes (Null clob     )))
+         (then (binary 0xAF) (denotes (Null blob     )))
+         (then (binary 0xBF) (denotes (Null list     )))
+         (then (binary 0xCF) (denotes (Null sexp     )))
+         (then (binary 0xDF) (denotes (Null struct   ))))
+
+(ion_1_1 "null values in Ion 1.1 binary"
+         (then (binary 0xEA)      (denotes (Null          )))
+         (then (binary 0xEB 0x00) (denotes (Null bool     )))
+         (then (binary 0xEB 0x01) (denotes (Null int      )))
+         (then (binary 0xEB 0x02) (denotes (Null float    )))
+         (then (binary 0xEB 0x03) (denotes (Null decimal  )))
+         (then (binary 0xEB 0x04) (denotes (Null timestamp)))
+         (then (binary 0xEB 0x05) (denotes (Null string   )))
+         (then (binary 0xEB 0x06) (denotes (Null symbol   )))
+         (then (binary 0xEB 0x07) (denotes (Null blob     )))
+         (then (binary 0xEB 0x08) (denotes (Null clob     )))
+         (then (binary 0xEB 0x09) (denotes (Null list     )))
+         (then (binary 0xEB 0x0A) (denotes (Null sexp     )))
+         (then (binary 0xEB 0x0B) (denotes (Null struct   ))))
+
+(ion_1_x "Null denotations"
```

Comment:
Do I understand correctly that this is checking whether the `denotes` operator is working as intended?

Reply:
I don't know. This was already part of the `nulls` file, so I kept it.